### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by lextirajoh**
Hey Luisa and Sander,

Please check out the navbar.

In the main.jsx and app.js there is also code for the routing and the conditional rendering of the navbar.

To be removed after approval:
- grey background
- vertical spacer (added so you can see the scroll effect = black  navbar when scrolled down)

Thanks  a lot.

### CODE REVIEW

**Comment by LuiWo** [comment]
Compliments on you for writing such neat codes.  
I feel like I got a better understanding of props after going through your code. 
Well done.

File: TG-Netflix/src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            <li>
+              <NavLink
+                to="browse"
+                style={({ isActive }) => (isActive ? activeStyle : undefined)}
+              >
+                Home
+              </NavLink>
+            </li>
+            <li>
+              <NavLink to="">Series</NavLink>
```
Comment (Sander-G) [inline TG-Netflix/src/components/navbar/Navbar.jsx:50]: NavLink naar series kan weg (?)

Comment (Sander-G) [reply TG-Netflix/src/components/navbar/Navbar.jsx:50]: idem voor 'new & popular' / 'my list' / 'browse by language' (?)

**Comment by LuiWo** [comment]
Review after changes. 

Codes are still nicely written.
Minor thing I would say is that the Navbar looks jumpy due to text size , icon and visibility changes that are happening at different breakpoint of each other. Could be because it's currently the main focus on the screen, but might change when more things are added. Can be a later tweak if need be.

As Sander mentioned above. The Home and films where the only ones we needed. (Unless it was changed now that there will be a proper login function that tracks the users. In this case My list should stay).

As for routing in regards to tracking who's logged in. A sign-out that leads to "Landing Page" would be nice. The rest of the routing can perhaps be added later depending how much deeper the user data tracking goes. 

Overall everything else seems to be working.

File: TG-Netflix/src/components/navbar/Navbar.jsx (hunk 1 above)
Comment (Sander-G) [reply TG-Netflix/src/components/navbar/Navbar.jsx:50]: deze waren nog niet zichtbaar ivm niet gesyncte commits :)

File: TG-Netflix/src/components/navbar/NavbarAccount.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            </li>
+            <li>
+              <a href="/account">Account</a>
+            </li>
+            <li>
+              <a href="#">Help Centre</a>
+            </li>
+            <hr />
+            <li>
+              <a href="#">Sign out of Netflix</a>
```
Comment (Sander-G) [inline TG-Netflix/src/components/navbar/NavbarAccount.jsx:37]: net als in 'standaard navbar' kan de Sign out link je weer terugbrengen naar de landingpage /

File: TG-Netflix/src/components/navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                    <RiAccountCircleLine size={20} />
+                    <a href="/account">Account</a>
+                  </li>
+                  <li>
+                    <BiHelpCircle size={20} />
+                    <a href="#">Help Centre</a>
+                  </li>
+                  <hr />
+                  <li>
+                    <a href="#">Sign out of Netflix</a>
```
[the remark this answers is not in the document]
Comment (lextirajoh) [reply TG-Netflix/src/components/navbar/Navbar.jsx:113]: Yep done.

File: TG-Netflix/src/components/navbar/NavbarAccount.jsx (hunk 2 above)
Comment (lextirajoh) [reply TG-Netflix/src/components/navbar/NavbarAccount.jsx:37]: Done.

**Review by Sander-G — APPROVED** [review]
Ziet er erg goed uit, Lex! In aanvulling wat Luisa aangeeft over de breakpoints en de verspringende text.. het origineel schaalt ook het logo kleiner op 1 van de breakpoints, ook hier zie je dat de tekst een beetje verspringt. Misschien is hier weinig aan te doen, aangezien het afhangt van de hoogte van het grootste element in de rij van de navbar flexbox container ( ik denk het logo of het icoon van de account dropdown), als het horizontaal gecentreerd is (het origineel doet het ook een beetje), het is ook alleen zichtbaar als je schermgrootte aanpast.

sander.

**Comment by lextirajoh** [comment]
> Review after changes. 

Codes are still nicely written.
Minor thing I would say is that the Navbar looks jumpy due to text size , icon and visibility changes that are happening at different breakpoint of each other. Could be because it's currently the main focus on the screen, but might change when more things are added. Can be a later tweak if need be.

As Sander mentioned above. The Home and films where the only ones we needed. (Unless it was changed now that there will be a proper login function that tracks the users. In this case My list should stay).

As for routing in regards to tracking who's logged in. A sign-out that leads to "Landing Page" would be nice. The rest of the routing can perhaps be added later depending how much deeper the user data tracking goes. 

Overall everything else seems to be working.  
 

Thanks for the feedback!

There's a lot happening in the navbar when resizing, but as far as I can see it mimics how it happens in the Netflix navbar. I personally don't think it is very elegant either.  They use no less than 7 breakpoints which is nuts. But if you see something specific which is not correct behavior let me know.

Just showing the Home and Films links. Good point. I had it like that in an earlier version. But after seeing Sander's and Esthers carbon copy work I decided to add the redundant nav links in. But perhaps for the navabr it is confusing. I'll check with the group what they prefer.

Thanks.

**Comment by lextirajoh** [comment]
@Sander

Dank voor je feedback!
Als je het hebt over de horizontale versprigning van de navlinkjes dan heb ik die er in mijn code net uitgehaald (op breakpoint 1150px). In NetFlix gebeurt dit maar het is lelijk om te zien en het voegt verder niks toe. Nu ziet het er wat vloeiender uit als je resized. Als je een ander breakpoint bedoelt hoor ik het wel.

**Pull request merged**